packages/material-ui/src/Collapse/Collapse.d.ts

```diff
@@ -4,10 +4,44 @@ import { Theme } from '../styles/createMuiTheme';
 import { TransitionProps } from '../transitions/transition';
 
 export interface CollapseProps extends StandardProps<TransitionProps, CollapseClassKey, 'timeout'> {
+  /**
+   * The content node to be collapsed.
+   */
   children?: React.ReactNode;
+  /**
+   * The height of the container when collapsed.
+   */
   collapsedHeight?: string | number;
+  /**
+   * The component used for the root node.
+   * Either a string to use a DOM element or a component.
+   */
   component?: React.ElementType<TransitionProps>;
-  theme?: Theme;
+  /**
+   * If `true`, the component will transition in.
+   */
+  in?: boolean;
+  /**
+   */
```

Comment:
Why the empty comments?

Reply:
To prevent documentation from the `TransitionProps`. It's basically a `@ignore`. See https://github.com/mui-org/material-ui/pull/20264#discussion_r397473964